addons/web/static/tests/fields/field_utils_tests.js

```diff
@@ -281,8 +282,8 @@ QUnit.test('parse percentage', function(assert) {
     assert.strictEqual(fieldUtils.parse.percentage(""), 0);
     assert.strictEqual(fieldUtils.parse.percentage("0"), 0);
     assert.strictEqual(fieldUtils.parse.percentage("0%"), 0);
-    assert.strictEqual(fieldUtils.parse.percentage("0.02"), 0.02);
-    assert.strictEqual(fieldUtils.parse.percentage("1"), 1);
+    assert.strictEqual(fieldUtils.parse.percentage("0.5"), 0.005);
+    assert.strictEqual(fieldUtils.parse.percentage("1"), 0.01);
     assert.strictEqual(fieldUtils.parse.percentage("2%"), 0.02);
```

Comment:
if you simplify the parsePercentage function, some asserts have to be removed or adapted here